input/http.md

```diff
@@ -22,11 +22,43 @@ For the full list of the configurable options, see the [Parameters](http.md#para
 
 ## Basic Usage
 
-Here is a simple example to post a record using `curl`, which uses the default Content-Type `application/x-www-form-urlencoded`
+By default, the data format depends on the `Content-Type`.
+In summary, you can send the follwoing format.
 
-```text
-# Post a record with the tag "app.log"
-$ curl -X POST -d 'json={"foo":"bar"}' http://localhost:9880/app.log
+* `json`
+* `ndjson`
+* `msgpack`
+
+Here is a simple example to post a record using `curl`, which uses the default Content-Type `application/x-www-form-urlencoded`.
+
+Example: post JSON data with the tag "app.log":
+
+```bash
+curl -X POST -d 'json={"foo":"bar"}' http://localhost:9880/app.log
+```
+
+Example: post NDJSON data with the tag "app.log":
+
+```bash
+ndjson=`echo -e 'ndjson={"k1":"v1"}\n{"k2":"v2"}\n'`
+curl -X POST -d "$ndjson" http://localhost:9880/app.log
+```
+
+Example: post MessagePack data with the tag "app.log":
+
+```bash
+msgpack=`echo -e "msgpack=\x81\xa3foo\xa3bar"`
+curl -X POST -d "$msgpack" http://localhost:9880/app.log
+```
+
+Some `Content-Type` other than `application/x-www-form-urlencoded` support specific formats.
+In that case, the format type specification as `json=` in the data is not necessary.
```

Comment:
> In that case, the format type specification as `json=` in the data is not necessary.

How about another wording something like: 

```
If media type is specified with `Content-Type: `, then content type prefix such as `json=` is not necessary for posting data.

Example: ....
```

Reply:
Thanks for your review!
Could tell me which points are your concerns?
I guess the aim is to make the explanation more concrete. Is this right?
If so, how about something like this?

```diff
- Some `Content-Type` other than `application/x-www-form-urlencoded` support specific formats.
- In that case, the format type specification as `json=` in the data is not necessary.
+ Some Media Types other than `application/x-www-form-urlencoded` support specific formats.
+ If those Media Types are specified with `Content-Type: `, the prefix such as `json=` is not necessary for posting data.
```

Reply:
> content type prefix such as `json=`

I have a concern about whether we can describe the prefix as `content type prefix`.
Isn't this prefix for the data format for parsing, rather than for `Content-Type`?

Reply:
> I guess the aim is to make the explanation more concrete. Is this right?

Yes. so https://github.com/fluent/fluentd-docs-gitbook/pull/502#discussion_r1630703860 suggested one seems good.

Reply:
OK! Thanks! I will fix it!

Reply:
Fixed!